src/VideoReceiver/VideoReceiver.pri

```diff
@@ -59,7 +60,6 @@ LinuxBuild {
 
         # Copy GStreamer plugins
         QMAKE_POST_LINK += $$escape_expand(\\n) xcopy \"$$GST_ROOT_WIN\\lib\\gstreamer-1.0\\*.dll\" \"$$DESTDIR_WIN\\gstreamer-plugins\\\" /Y $$escape_expand(\\n)
-        QMAKE_POST_LINK += $$escape_expand(\\n) xcopy \"$$GST_ROOT_WIN\\lib\\gstreamer-1.0\\validate\\*.dll\" \"$$DESTDIR_WIN\\gstreamer-plugins\\validate\\\" /Y $$escape_expand(\\n)
```

Comment:
@andrewvoznytsa why was this gst-validate copied to the installer? Where is it used? It does not exist in the gstreamer 1.18 package, and the build/program does not break after removing this line.

Reply:
@jaxxzer no idea. git blame (good thing, BTW) says that this question goes to @DonLakeFlyer (https://github.com/mavlink/qgroundcontrol/commit/bb04b28405b7189de880c203445889225d716e7a)